tests/zap.py

```diff
@@ -103,11 +106,11 @@ class Main:
         summary.add_row(["Low", low])
         summary.add_row(["Medium", medium])
         summary.add_row(["High", high])
-        print(summary)
+        logger.info(summary)
 
         for url in sort_by_url:
-            print()
-            print(url)
+            logger.info()
+            logger.info(url)
```

Comment:
I don't think we can use `logger.info` this way since it requires a positional argument. Perhaps something like this:

```suggestion
            logger.info("\n" + url)
```